client/store.js

```diff
@@ -244,6 +254,12 @@ store.setupKresus = function(cb) {
         has(world, 'accounts');
         has(world, 'operations');
 
+        let defaultAccountId = store.getDefaultAccountId();
+        if (defaultAccountId && !data.currentAccountId)
+        {
+            data.currentAccountId = defaultAccountId;
```

Comment:
OK, I just saw that you're also setting the data.currentBankId.

How about:
1. We retrieve the `defaultAccountId` here, but don't set `data.currentAccountId`.
2. The if below gets replaced by `if (acc.id === defaultAccountId) { data.currentAccountId = acc.id; data.currentBankId = bank.id; }`
3. After settings accounts, `assert(defaultAccountId.length > 0 && data.currentAccountId === defaultAccountId);`

What do you think?